cmd/logs/install/cmd.go

```diff
@@ -82,11 +82,10 @@ func run(cmd *cobra.Command, argv []string) {
 
 	// Allow the command to be called programmatically
 	if len(argv) == 1 && !cmd.Flag("cluster").Changed {
-		clusterKey = argv[0]
+		ocm.SetClusterKey(argv[0])
 		watch = true
-	} else {
-		clusterKey = r.GetClusterKey()
 	}
+	clusterKey = r.GetClusterKey()
```

Comment:
we can get rid of the `var clusterKey string` above here (and in the other file) as well

Reply:
Removed.